packages/react-core/src/components/FormSelect/examples/FormSelectInvalid.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { FormSelect, FormSelectOption, ValidatedOptions } from '@patternfly/react-core';
+
+export const FormSelectInvalid: React.FunctionComponent = () => {
+  const [formSelectValue, setFormSelectValue] = React.useState('');
+  const [validated, setValidated] = React.useState<ValidatedOptions>(ValidatedOptions.default);
+
+  const isEmpty = () => formSelectValue === '';
+  const onChange = (value: string) => {
+    setFormSelectValue(value);
+  };
+
+  React.useEffect(() => {
+    const validationState = !isEmpty() && ValidatedOptions.error;
```

Comment:
This line just needs to be updated so that it sets the validated state correctly; right now it's having the opposite effect of a valid selection rendering the error icon in the example.

```suggestion
    const validationState = isEmpty() && ValidatedOptions.error;
```

Reply:
Okay so iiuc, only `Select a number` option should be invalid? If this is the case, won't it be better to use e.g. 3 as the only valid option and add to corresponding `FormSelectOption`s label that this is the only valid option? Similarly as in `Validated` example.

Reply:
> Okay so iiuc, only Select a number option should be invalid?

That's what it seemed like the intention originally was, at least based on the variable being named `isEmpty`.

> If this is the case, won't it be better to use e.g. 3 as the only valid option and add to corresponding FormSelectOptions label that this is the only valid option? Similarly as in Validated example.

I think either way could work really. While both Validated and Invalid examples are very similar, the main difference just seems to be that the Validate example shows various states of validation instead of only an error like the Invalid example does. As long as that remains true, reusing the `onChange` from the Validate example and tweaking it to not use helper text or simulate a network call could work (this would also make the useEffect unnecessary)

Reply:
Okay, so I kept the original example, it does what we need it to do so it's probably not necessary to make it more complicated.